builtin/logical/cassandra/backend_test.go

```diff
@@ -35,7 +35,7 @@ func TestBackend_roleCrud(t *testing.T) {
 		Backend:        b,
 		Steps: []logicaltest.TestStep{
 			testAccStepConfig(t),
-			testAccStepRole(t),
+			testAccStepRoleWithOptions(t),
```

Comment:
Sadly Cassandra doesn't yet use `dockertest` so these tests are skipped by Travis -- but this will actually cause tests to fail. You removed the role creation step that writes the role `test` in favor of one that writes the role `test2`, so when the next steps are executed it'll fail. Probably you should do both -- call `testAccStepRole` and the rest using `"test"`, then call the options version and the rest using "test2".